server/src/main/java/org/elasticsearch/common/metrics/MeanMetric.java

```diff
@@ -20,13 +20,10 @@ public void inc(long n) {
         sum.add(n);
     }
 
-    public void dec(long n) {
-        counter.decrement();
-        sum.add(-n);
-    }
-
     public long count() {
```

Comment:
can you add a comment why count is supposed to only return `>= 0`

Reply:
added in https://github.com/elastic/elasticsearch/pull/71446/commits/cb76a362fbd6fa603d91ad91c0ab8ec6938b797f